musica/micm/micm.meta

```diff
@@ -0,0 +1,59 @@
+[ccpp-table-properties]
+  name = micm
+  type = scheme
+
+[ccpp-arg-table]
+  name  = micm_init
+  type  = scheme
+[ filepath ]
+  standard_name = enter_standard_name_3
```

Comment:
CCPP standard names, units, etc. need to be added.  I don't know if you want to do this before this PR is brought in or not.  It certainly won't run until this is all filled in

Reply:
The filepath will be a path to a configuration file needed by micm. I'm not sure what name that should have or if this should be passed by some other code in CAMDEN that wraps around this.

Reply:
The filepath will be a path to a configuration file needed by micm. I'm not sure what name that should have or if this should be passed by some other code in CAMDEN that wraps around this.

Reply:
@cacraigucar I have added a PR into CCPPStandardNames to add a filepath for micm: https://github.com/ESCOMP/CCPPStandardNames/pull/40

Reply:
and I believe I have corrected the standard names to match what is there and changed the code appropriately

Reply:
Resolving this as I can confirm all necessary CCPP metadata is now contained within this PR.